### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by ogagaoghene**
# OOP school library: add basic UI

## Description 
An interactive console based school library app written in Ruby to demonstrate the concept Object Oriented Programming(OOP). Two files - main.rb and app.rb were added to setup the program's entry point - the point of execution of a program.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 # Project Name
 
-> Description the project.
-A school library application written in Ruby. Classroom, rentals and books classes are added. Relationships such as one to many, belongs_to, one to one relationships are created between the different classes.
+> Description of the project.
+An interactive console based school library app written in Ruby to demonstrate the concept Object Oriented Programming(OOP). Two files - main.rb and app.rb were added to setup the program's entry point - the point of execution of a program.  
+
 
 ## Built With
```
Comment (fobadara) [inline README.md:10]: [OPTIONAL] Good job including all the necessary header for your README but if I didn't have prior knowledge of Ruby I wouldn't have known what to do with your app. Kindly tell us who to run it. You can check this [repo ](https://github.com/fobadara/quest-student-library/blob/development/README.md)out for hints

[OPTIONAL] Your MIT license is taken us to a 404 page

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+  # List all books
+  def list_all_books
+    if @books.empty?
+      puts 'Book list is empty'
+    else
+      puts 'List of all Books'
+      @books.each { |book| puts "Title: #{book.title} Author: #{book.author}" }
+    end
+  end
```
Comment (fobadara) [inline app.rb:28]: [OPTIONAL] I think you should try to catch errors in your code. For example, in places where you are expected to enter. number but you enter letters. You should find ways to warn the user and continue or return back. to the initial list. But your code just gives an error and stops running.

**Review by fobadara — APPROVED** [review]
Hi @ogagaoghene, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters are passing ✅ 
- Good PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (ogagaoghene) [reply app.rb:28]: Thanks for your professional code review. I will do as advised subsequently.

File: README.md (hunk 1 above)
Comment (ogagaoghene) [reply README.md:10]: Thanks for pointing out this repo as a useful reference. I appreciate you greatly

**Comment by ogagaoghene** [comment]
Many thanks to you for your professional code review

**Pull request merged**